integration-tests/playbook_verifier/test_verifier.py

```diff
@@ -19,15 +30,34 @@
         "compliance_openscap_setup.yml",
     ],
 )
-def test_official_playbook(filename: str):
-    """insights-client contains playbook verifier application.
-
-    It is used by rhc-worker-playbook and rhc-worker-script to safely deliver
-    and run Red Hat signed playbooks.
-
-    In this test, the official playbooks are verified against the GPG key
-    the application ships.
+@pytest.mark.tier1
+def test_official_playbook(insights_client, filename: str):
+    """
+    :id: 3659e27f-3621-4591-b1c4-b5f0a277bb72
+    :title: Test playbook verifier
+    :description:
+        This test verifies the official playbooks against the GPG key
+        the application ships.
+    :tags: Tier 1
+    :steps:
+        1. Read playbook file content
+        2. Run insights-client verifier with playbook
+        3. Compare output to input
+    :expectedresults:
+        1. File content is correctly read and loaded into memory
+        2. Subprocess executes successfully without errors
+        3. Verifier's output matches original playbook content
     """
+    if (
+        sys.version_info >= (3, 12)
+        and insights_client.core_version < Version(3, 5, 2)
+        and filename == "bugs.yml"
+    ):
+        pytest.xfail(
+            f"Core {insights_client.core_version} suffers from "
+            "CCT-1065, CCT-1101, CCT-1102."
+        )
```

Comment:
I think we don't have to do this in `el8` branch. The test file `bugs.yml` was only aimed for RHEL10 (see CCT-1070) and also Python3.12 is not default for RHEL8.